modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Fields.tsx

```diff
@@ -291,6 +291,7 @@ const SaveFDSFieldsModalContent = ({
 												}))
 											)
 										}
+										name="custom-control-input-checkbox"
```

Comment:
Why exactly was this added?

Reply:
Hey @bryceosterhaus, according to [LPS-183320](https://liferay.atlassian.net/browse/LPS-183320), it was missing a label which was shown when running axe devTools.

Reply:
Yeah I see that, but why did you use the name of "custom-control-input-checkbox"?

Reply:
I named it as such because "custom-control-input" is the class name of the checkbox.